Lab2.ipynb

```diff
@@ -0,0 +1,805 @@
+      "text/plain": [
+       "<Figure size 1800x504 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "poisson_plots(avg=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cd7837ed-5ec4-43c0-9ecf-c20b54b27afe",
+   "metadata": {},
+   "source": [
+    "### D)\n",
+    "Choosing 5 as my number of days, I would see 40 gamma rays."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "47026a92-79ca-46fb-ba09-d9f446a1cbe4",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "7.341405446132749e-13\n"
+     ]
+    }
+   ],
+   "source": [
+    "k = np.linspace(0,20,21)\n",
+    "poisson = stats.poisson.pmf(k,mu=CR_Background)\n",
+    "\n",
+    "dist = convolve(poisson,n=5)\n",
+    "probability = sum(dist[40:])\n",
+    "print(probability)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "baa708e7-7fc6-4f0d-9923-014a65d152a3",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "7.077443434036951"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "stats.norm.ppf(1-probability)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "88cacde0-0294-4f84-bcac-1ac65f674466",
+   "metadata": {},
+   "source": [
+    "So I see that this cooresponds to a likelihood of ~7 sigmal; very significant.\n",
+    "\n",
+    "## Problem 2\n",
+    "\n",
+    "### A)"
```

Comment:
In this part it asks you show how the distribution changes as you average over more intervals. I think this means it wants you to plot the averages of each convolution? I'm not sure if this is the correct interpretation but I think that is why our plots are so different, because I clamp the values to the original domain.